tests/e2e/service_invocation/service_invocation_test.go

```diff
@@ -231,11 +231,11 @@ func TestHeaders(t *testing.T) {
 		assert.Equal(t, "application/json", requestHeaders["Content-Type"][0])
 		assert.Equal(t, "DaprValue1", requestHeaders["Daprtest-Request-1"][0])
 		assert.Equal(t, "DaprValue2", requestHeaders["Daprtest-Request-2"][0])
-		assert.NotNil(t, requestHeaders["Forwarded"][0])
 		assert.NotNil(t, requestHeaders["Traceparent"][0])
 		assert.NotNil(t, requestHeaders["User-Agent"][0])
 		assert.NotNil(t, requestHeaders["X-Forwarded-For"][0])
-		assert.Equal(t, "http", requestHeaders["X-Forwarded-Proto"][0])
+		assert.NotNil(t, requestHeaders["X-Forwarded-Host"][0])
```

Comment:
Can we check the exact host name instead of checking nil?

Reply:
That's better. Changed it to check values.